database/metadata/databases/default/tables/public_people_list_view.yaml

```diff
@@ -2,6 +2,15 @@ table:
   name: people_list_view
   schema: public
 object_relationships:
```

Comment:
adds a manual gql relationship between the people list view and crashes so that we can query related crash data through a person-level record